src/wp-includes/class-wp-autoloader.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class WP_Autoloader
+ *
+ * WordPress Class, Interface and Trait Autoloader
+ *
+ */
+class WP_Autoloader {
+
+	/**
+	 * @var array
+	 */
+	private static $_classmap;
+
+	/**
+	 * WP_Autoloader constructor.
+	 */
+	function __construct() {
+
+		if ( ! defined( 'DISABLE_CORE_AUTOLOADER' ) ) {
+
+			spl_autoload_register( array( $this, 'load' ), true, true );
+
+			if ( is_file( ABSPATH . 'wp-classmap.php' ) ) {
```

Comment:
You should include the `spl_autoload_register()` call within this condition only. Otherwise, each class will run through an invalid autoloader if the file could not be loaded.